integration/registry_create_test.go

```diff
@@ -0,0 +1,90 @@
+package integration
```

Comment:
really appreciate you writing an integration test 💃

Reply:
of course!